src/__tests__/extensions/toolbar.js

```diff
@@ -158,4 +161,32 @@ describe('Toolbar', () => {
             expect(given.subject()).toBe(false)
         })
     })
+
+    describe('afterDecideResponse', () => {
+        given('localStorage', () => ({
+            setItem: jest.fn(),
+            removeItem: jest.fn(),
+            getItem: jest.fn(() => null),
+        }))
+
+        it('should enable the toolbar based on the /decide response', () => {
+            given('decideResponse', () => ({ isAuthenticated: true, editorParams: { toolbarVersion: 'toolbar' } }))
+            given.toolbar.afterDecideResponse(given.decideResponse, given.localStorage)
+            expect(given.localStorage.setItem).toHaveBeenCalledWith('toolbar_enabled_by_user', '1')
+        })
+
+        it('should disable the toolbar based on the /decide response', () => {
+            given('decideResponse', () => ({}))
+            given.toolbar.afterDecideResponse(given.decideResponse, given.localStorage)
+            expect(given.localStorage.setItem).not.toHaveBeenCalled()
+            expect(given.localStorage.removeItem).toHaveBeenCalledWith('toolbar_enabled_by_user')
+        })
+
+        it('should disable the toolbar based on the /decide response', () => {
+            given('editorParams', () => ({ toolbarVersion: 'toolbar' }))
```

Comment:
Not explicit that the `/decide` response returned a disabled mode, but think it's set this way anyways.